### DISCUSSION TITLE
Error in slot(object = object, name = layer) : argument "slot" is missing, with no default

### DISCUSSION THREAD

**Discussion by echo1899** [stuart-lab/signac#1558] [Q&A] [answered]
I have a question about 'TSSEnrichment' function. Last week I was able to run it successfully, but now when I try I get the following error:
_Error in slot(object = object, name = layer) :  argument "slot" is missing, with no default_

I tried downloading the data and following the tutorial in case the issue is with my object (https://github.com/stuart-lab/signac/blob/HEAD/vignettes/pbmc_vignette.Rmd), but I am still getting the error and I can't find any information on it.

**Comment by timoast** [comment, marked as answer]
Please install the latest version from the develop branch:

```r
remotes::install_github("stuart-lab/signac", ref = "develop")
```

**Reply by echo1899** [reply]
That worked, thank you!
